include/natalie/method_info.hpp

```diff
@@ -30,9 +30,9 @@ class MethodInfo {
 
     bool is_defined() const { return !m_undefined && m_method; }
 
+    void visit_children(Cell::Visitor &);
```

Comment:
This would probably need a `virtual` and `override` added to it (although I'm not really sure what happens without them).

Reply:
MethodInfo doesn't subclass from Cell, so it isn't virtual in this case.